### ISSUE TITLE
The ToolStripButton borders have low color contrast ratio in high contrast mode

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#6220]
NET Core Version: 
- .NET Core SDK: 7.0.0-alpha.1.21567.1

Have you experienced this same bug with .NET Framework?: 
- Yes for `ToolStripRenderMode.System` (from #5502)
- No for other rendered modes

**Problem description:**
1. Create a `Form` with a `ToolStrip`
2. Add a `ToolstripButton` to `ToolStrip`
3. Set `System`  rendering mode
4. Enable high contrast mode
5. Run the application

**Actual behavior:**
High Contrast Black:
![image](https://user-images.githubusercontent.com/23376742/143254853-8a9d01e2-55a2-49c8-bd9c-fc4834e8e82c.png)

High Contrast White:
![image](https://user-images.githubusercontent.com/23376742/143257200-a37d04f6-88de-43cd-a050-9879f5651824.png)

**Expected behavior:**
High Contrast Black:
![image](https://user-images.githubusercontent.com/23376742/143255044-f25d0067-0c8e-4ba3-a1df-80089599fb8f.png)

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw  - could you please remind me if you recommended to port this one to NETFX or not? AI will detect.

**Comment by merriemcgaw** [comment]
If we have bandwidth I'd probably recommend it.

**Comment by Tanya-Solyanik** [comment]
@Amy-Li03, @vladimir-krestov  - I think we fixed this bug in 4.8.1 - https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1367573, but had not ported to .NET7. If this is the case, Vladimir, please prioritize this issue

**Comment by Amy-Li03** [comment]
@Tanya-Solyanik verified this bug in 4.8.1 on win10 OS after installed latest .Net framework 4.8.1: \\\cpvsbuild\drops\dev14\NetFXDev1\layouts\amd64ret\04349.00\enu\netfx\fulloob\oob, bug is still reproducing, the test result is same as above screenshot in Actual behavior.

**Comment by dkazennov** [comment]
Current results in 7.0.0-preview.4.22229.4. 

![image](https://user-images.githubusercontent.com/102961955/172819435-edaa3397-702b-4f47-9bdf-793f772d0691.png)
![image](https://user-images.githubusercontent.com/102961955/172819531-5b2754af-e16b-4a94-8888-652d3a4a046c.png)

**Comment by dkazennov** [comment]
It seems that this issue is similar with https://github.com/dotnet/winforms/issues/5502

**Comment by dkazennov** [comment]
After the investigation and consultations I believe that this task should be closed.

1. We can not affect what color was been selected in Windows settings. We can just use different color scheme properties (like Color.Highlight) for different components of Winforms.
2. The doted line rectangle was introduces in https://github.com/dotnet/winforms/pull/5614 (it is related to https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1367573). It is being drawn over colored elements whose colors are chosen in Windows settings.
3.  We can add some overlaying graphics on the Winforms layer just like in PR 5614, but there is no color that be both equally contrast with the form background black color and the turquoise background color of a checked button.
4. We should not use doted line since it is already reserved for a selected button.
5. It appears there is no solution for this issue in .Net Framework 4.8.1
6. The turquoise background color of a checked button gives enough contrast to separate it from the background, isn't it?

**Comment by dkazennov** [comment]
Some additional information.
It is possible to add overlaying graphics with custom colors line in case of https://github.com/dotnet/winforms/pull/6557 with the help of ButtonBaseAdapter class and it derivates or with creating new instances of Pen and SolidBrush classes. 

But it doesn't help with the fact that if a border rectangle is contrast with form background it won't be contrast with checked button background. E.g. if we change grey borders (192, 192, 192) with black borders, the borders will be contrast with white form background but not with dark checked button background. Slight tone changes don't fix the Issue either.

![image](https://user-images.githubusercontent.com/102961955/173601434-664c566b-99df-4f5f-b81a-a7f3a3b86d38.png)

**Comment by merriemcgaw** [comment]
I think you're probably right. The button needs to have the correct contrast with the text. The border is of less importance per the official CAIAcc team. So we can go ahead and close this issue.
